src/@chakra-ui/gatsby-plugin/styles.ts

```diff
@@ -0,0 +1,12 @@
+const styles = {
+  global: (_props) => ({
+    // TODO: remove these overrides as we adopt the new Design System
```

Comment:
A bit more documentation/context on why this is needed to begin with would be helpful.

Reply:
e.g. why is both this file & GlobalStyle.tsx needed?